packages/fether-react/src/Accounts/AccountsList/AccountsList.js

```diff
@@ -72,6 +72,7 @@ class AccountsList extends Component {
                       className='-clickable'
                       type={accountsInfo[address].type}
                       name={accountsInfo[address].name || '(no name)'}
+                      screen='accounts'
```

Comment:
This prop only decides if the account info is justified on the left or in the center, right? In this case, can we rename this prop to `justify='left' | 'center'`, or `isCentered={true|false}`? It seems weird that a prop says on which screen the component is located.

Reply:
There is only hard-coded styles deciding if the account info is justified left or center.
There is an `account -header` style that's used here for the header on the "Account page" https://github.com/paritytech/fether/blob/5d620bed8f0d55d00fc678cfa4c0cabadf7550a0/packages/fether-ui/src/AccountHeader/AccountHeader.js#L22. We use the className `account -header` so it uses justify center as defined here https://github.com/paritytech/fether/blob/5d620bed8f0d55d00fc678cfa4c0cabadf7550a0/packages/fether-react/src/assets/sass/components/_account.scss#L5, otherwise if it were just `account` (as we use on the "Account List page" then it's justified to the left by default (but we don't have to write it explicitely.

The prop `screen='accounts'` that you mentioned is actually passed down to the Name component associated with the AccountCard since it's used both in the body and header components of the Fether window. The "Accounts List page" is the only page that has a top and bottom margin that makes the Information component appear good (which contains the Name and Address component). However other pages that it's used are in the header area, but have insufficient top and bottom margin, as shown below:

<img width="279" alt="screen shot 2019-01-23 at 12 29 15 am" src="https://user-images.githubusercontent.com/6226175/51538663-0479b400-1ea6-11e9-9e20-9cdca4aa5ac1.png">

So in the Name component I've use this logic https://github.com/paritytech/fether/pull/363/files#diff-1f4b2128964b51e8a48e4ee7c84a3e9fR13 so it applies the className `account_name -header` if the Name component is being used in the header area (which applies some margin above and below the name so the overall Information component in the header appears the same as where it's used on the "Accounts List page" https://github.com/paritytech/fether/pull/363/files#diff-1d583cfb15e661b39f28b9b82fa329c0R47) as shown below:

<img width="361" alt="screen shot 2019-01-23 at 12 25 57 am" src="https://user-images.githubusercontent.com/6226175/51538516-97661e80-1ea5-11e9-92d0-2494e8dda77d.png">

<img width="361" alt="screen shot 2019-01-23 at 12 27 07 am" src="https://user-images.githubusercontent.com/6226175/51538537-a77dfe00-1ea5-11e9-808f-9e1137746c1d.png">

Otherwise it just uses the className `accounts_name`.

Reply:
I understand, my proposal was to rename the `screen` props to `justify`, which I think describes better the intent:
```
className={`account_name ${justify === 'left' ? '-header' : ''}`}
```

if you think `screen` name fits better, then we can also keep it.

Reply:
> I understand, my proposal was to rename the `screen` props to `justify`, which I think describes better the intent:
> 
> ```
> className={`account_name ${justify === 'left' ? '-header' : ''}`}
> ```
> if you think `screen` name fits better, then we can also keep it.

I understand what you're saying, but I don't think I made myself clear enough in my last message. The ternary logic that I've added to the `Name` component only determines whether to apply some top/bottom padding to the `Name` component, (**not** to apply any left/center justification), so I don't think `justify` is the right word, since I'm **not** using it to justify any left/right alignment.

I think meaningful alternatives to the word `screen` would be `isPageWithoutCustomVerticalPadding` or similar